--- xine-ui-0.99.5/src/xitk/skins/xinetic/Makefile.am.orig	2009-10-29 05:00:05.532407118 +0100
+++ xine-ui-0.99.5/src/xitk/skins/xinetic/Makefile.am	2009-10-29 05:00:29.030792127 +0100
@@ -15,7 +15,6 @@
 	CtlSave.png	   MainNext.png	      MrlDismiss.png	PlPlay.png	Letters.png       \
 	MainPause.png      MrlSelect.png      Letters2.png      MrlFlit.png                       \
 	MainAutoButton.png MainPlay.png	      Onglet.png	MainNav.png                       \
-	MainAutoButton.png MainPlay.png	      Onglet.png	                                  \
 	BigArrowLeft.png  BigArrowRight.png                                                       \
 	PlSlidHBg.png      SliderHPad.png     SkSlidHBg.png     SlidHPad.png	                  \
 	SkSlidHPad.png     skinconfig
