--- xine-ui-0.9.21/configure.ac.orig	2003-05-20 21:12:06.000000000 +0200
+++ xine-ui-0.9.21/configure.ac	2003-05-20 21:14:38.000000000 +0200
@@ -824,7 +824,7 @@
 dnl
 dnl Some include paths ( !!! DO NOT REMOVE !!! )
 dnl
-INCLUDES='-I$(top_srcdir) -I$(top_builddir) -I$(top_srcdir)/readline -I$(top_builddir)/readline -I$(top_srcdir)/src -I$(top_builddir)/src -I$(top_srcdir)/src/common -I$(top_builddir)/src/common -I$(top_srcdir)/src/xitk/xine-toolkit -I$(top_builddir)/src/xitk/xine-toolkit -I$(prefix)/include $(INTLDIR)'
+INCLUDES='-I$(top_srcdir) -I$(top_builddir) -I/usr/include/readline -I$(top_srcdir)/src -I$(top_builddir)/src -I$(top_srcdir)/src/common -I$(top_builddir)/src/common -I$(top_srcdir)/src/xitk/xine-toolkit -I$(top_builddir)/src/xitk/xine-toolkit -I$(prefix)/include $(INTLDIR)'
 AC_SUBST(INCLUDES)
 
 AC_DEFINE(NEED_MRLBROWSER,,[Define this to add mrl browser into xitk lib])
--- xine-ui-0.9.21/Makefile.am.orig	2003-03-25 15:33:01.000000000 +0100
+++ xine-ui-0.9.21/Makefile.am	2003-05-20 21:13:58.000000000 +0200
@@ -6,7 +6,7 @@
 
 ACLOCAL_AMFLAGS = @XINE_ACFLAGS@ -I m4
 
-SUBDIRS = doc m4 readline intl po misc src  
+SUBDIRS = doc m4 intl po misc src  
 
 DEBFILES = debian/README.Debian debian/changelog debian/control \
 	debian/copyright debian/links debian/menu \
--- xine-ui-0.9.21/src/xitk/Makefile.am.orig	2003-03-13 20:10:33.000000000 +0100
+++ xine-ui-0.9.21/src/xitk/Makefile.am	2003-05-20 21:15:20.000000000 +0200
@@ -48,11 +48,10 @@
 	     $(X_LIBS) $(XTEST_LIBS) $(CURL_LIBS) -lX11 -lm $(DBG_LIBS)
 
 xine_remote_SOURCES = xine-remote.c
-xine_remote_DEPENDENCIES = $(top_builddir)/readline/libreadline.la \
-			   $(top_builddir)/src/common/libcommon.la \
+xine_remote_DEPENDENCIES = $(top_builddir)/src/common/libcommon.la \
 			   xine-remote.c
 xine_remote_LDADD = $(NET_LIBS) $(THREAD_LIBS) \
-	$(top_builddir)/readline/libreadline.la $(TERMCAP_LIB) \
+	-lreadline $(TERMCAP_LIB) \
 	$(top_builddir)/src/common/libcommon.la $(DBG_LIBS)
 
 session_ctrl_LDADD = @XINE_LIBS@ $(top_builddir)/src/common/libcommon.la $(DBG_LIBS)
@@ -87,9 +86,6 @@
 	videowin.h \
 	viewlog.h
 
-$(top_builddir)/readline/libreadline.la:
-	@cd $(top_builddir)/readline && $(MAKE) libreadline.la
-
 $(top_builddir)/src/common/libcommon.la:
 	@cd $(top_builddir)/src/common && $(MAKE) libcommon.la
 
