diff -durN xine-ui-0.9.12.orig/configure.in xine-ui-0.9.12/configure.in
--- xine-ui-0.9.12.orig/configure.in	Sun Jun 23 15:46:04 2002
+++ xine-ui-0.9.12/configure.in	Sun Jun 23 15:48:18 2002
@@ -697,7 +697,6 @@
 readline/Makefile
 src/Makefile
 src/common/Makefile
-src/corba/Makefile
 src/Imlib-light/Makefile
 src/xitk/Makefile
 src/xitk/xine-toolkit/Makefile
diff -durN xine-ui-0.9.12.orig/src/Makefile.am xine-ui-0.9.12/src/Makefile.am
--- xine-ui-0.9.12.orig/src/Makefile.am	Sat Feb  2 22:12:35 2002
+++ xine-ui-0.9.12/src/Makefile.am	Sun Jun 23 15:48:07 2002
@@ -1,5 +1,5 @@
 
-SUBDIRS = common Imlib-light corba xitk aaui dfb
+SUBDIRS = common Imlib-light xitk aaui dfb
 
 debug:
 	@list='$(SUBDIRS)'; for subdir in $$list; do \
