diff -urN xine-ui-0.5.3/configure.in xine-ui-0.5.3.new/configure.in
--- xine-ui-0.5.3/configure.in	Wed Sep  5 03:13:28 2001
+++ xine-ui-0.5.3.new/configure.in	Wed Sep 12 19:16:21 2001
@@ -305,7 +305,7 @@
     ;;
   *)
     dnl Set the appropriate architecture define
-    case "$host_alias" in
+    case "$host" in
     i?86-* | k?-* | athlon-*) dnl catch i386,i486,i586,i686,k6,k7
 
         dnl Check for gcc cpu optimization support
@@ -328,7 +328,7 @@
         dnl enable x86 specific parts of the code
 
 	if test x"$sarchopt" != "xno"; then
-	    [case "$host_alias" in
+	    [case "$host" in
 		    i386-*) # *BSD return this even on a P III #-))
                       GLOBAL_CFLAGS="$GLOBAL_CFLAGS $sarchopt=i386"
                       DEBUG_CFLAGS="$DEBUG_CFLAGS $sarchopt=i386"
