--- xine-ui-0.9.7/src/xitk/skins/Makefile.am~	Mon Nov 26 19:37:54 2001
+++ xine-ui-0.9.7/src/xitk/skins/Makefile.am	Sun Dec 23 20:31:00 2001
@@ -1,12 +1,12 @@
 SUBDIRS = lcd pitt xinetic
 
 install-exec-am:
-	if test -d @XINE_SKINDIR@/metal; then \
-	  rm -rf @XINE_SKINDIR@/metal; \
+	if test -d $(DESTDIR)@XINE_SKINDIR@/metal; then \
+	  rm -rf $(DESTDIR)@XINE_SKINDIR@/metal; \
 	fi
-	if test -d @XINE_SKINDIR@/default; then \
-	  rm -rf @XINE_SKINDIR@/default && \
-	  $(LN_S) @XINE_SKINDIR@/xinetic @XINE_SKINDIR@/default; \
+	if test -d $(DESTDIR)@XINE_SKINDIR@/default; then \
+	  rm -rf $(DESTDIR)@XINE_SKINDIR@/default && \
+	  $(LN_S) $(DESTDIR)@XINE_SKINDIR@/xinetic $(DESTDIR)@XINE_SKINDIR@/default; \
 	fi
 
 debug:
