--- xine-ui-0.9.23/src/xitk/download.c.orig	2004-04-12 21:48:45.855184848 +0200
+++ xine-ui-0.9.23/src/xitk/download.c	2004-04-12 21:49:00.811911080 +0200
@@ -96,24 +96,24 @@
     
     curl_easy_setopt(curl, CURLOPT_VERBOSE, 
 #ifdef DEBUG
-		     TRUE
+		     1
 #else
-		     FALSE
+		     0
 #endif
 		     );
     
     curl_easy_setopt(curl, CURLOPT_URL, url);
     
     curl_easy_setopt(curl, CURLOPT_USERAGENT, user_agent);
-    curl_easy_setopt(curl, CURLOPT_NOSIGNAL, TRUE);
-    curl_easy_setopt(curl, CURLOPT_FAILONERROR, TRUE); 
-    curl_easy_setopt(curl, CURLOPT_NETRC, TRUE);
-    curl_easy_setopt(curl, CURLOPT_FOLLOWLOCATION, TRUE);
+    curl_easy_setopt(curl, CURLOPT_NOSIGNAL, 1);
+    curl_easy_setopt(curl, CURLOPT_FAILONERROR, 1); 
+    curl_easy_setopt(curl, CURLOPT_NETRC, 1);
+    curl_easy_setopt(curl, CURLOPT_FOLLOWLOCATION, 1);
 
     curl_easy_setopt(curl, CURLOPT_WRITEFUNCTION, store_data);
     curl_easy_setopt(curl, CURLOPT_WRITEDATA, (void *)download);
 
-    curl_easy_setopt(curl, CURLOPT_NOPROGRESS, FALSE);
+    curl_easy_setopt(curl, CURLOPT_NOPROGRESS, 0);
     curl_easy_setopt(curl, CURLOPT_PROGRESSFUNCTION, progress_callback);
     curl_easy_setopt(curl, CURLOPT_PROGRESSDATA, (void *)download);
     
