--- configure.ac	2007-05-04 02:35:18.755421561 +0200
+++ configure.ac-new	2007-05-04 02:40:20.610440809 +0200
@@ -304,7 +304,7 @@
    AS_HELP_STRING([--without-ncurses], [Turn off ncurses support.]))
 
 if test "x$with_ncurses" != "xno"; then
-  AC_CHECK_HEADERS(curses.h)
+  AC_CHECK_HEADERS(ncurses/ncurses.h)
   AC_CHECK_LIB(ncurses, cbreak,
                AC_DEFINE(HAVE_NCURSES,,[Define this if you have ncurses installed])
                ac_have_ncurses="yes"
