diff -urN xine-ui-0.9.4/src/xitk/skins/lcd/Makefile.am xine-ui-0.9.4.new/src/xitk/skins/lcd/Makefile.am
--- xine-ui-0.9.4/src/xitk/skins/lcd/Makefile.am	Tue Nov  6 23:53:01 2001
+++ xine-ui-0.9.4.new/src/xitk/skins/lcd/Makefile.am	Wed Nov  7 00:10:16 2001
@@ -10,7 +10,7 @@
 	mrl.png pl_inputtxt.png pl_play.png pl_delete.png pl_deleteall.png \
 	pl_moveup.png pl_movedown.png ctl_bg.png mrl_bg.png mrl_play.png \
 	playslow.png playfast.png volumebg.png volumepad.png mute.png cam.png \
-	sk_slidv.png sk_itembtn.png skinconfig
+	skinconfig
 
 debug:
 install-debug: install
diff -urN xine-ui-0.9.4/src/xitk/skins/metal/Makefile.am xine-ui-0.9.4.new/src/xitk/skins/metal/Makefile.am
--- xine-ui-0.9.4/src/xitk/skins/metal/Makefile.am	Tue Nov  6 23:53:01 2001
+++ xine-ui-0.9.4.new/src/xitk/skins/metal/Makefile.am	Wed Nov  7 00:10:28 2001
@@ -9,8 +9,7 @@
 	right.png slider_bg.png slider_pad.png slider_v.png smallbtn.png \
 	stop.png playlist.png setup.png mrl.png pl_play.png pl_delete.png \
 	pl_deleteall.png pl_moveup.png pl_movedown.png pl_inputtxt.png \
-	playslow.png playfast.png mrl_play.png mute.png cam.png sk_itembtn.png \
-	sk_slidv.png skinconfig
+	playslow.png playfast.png mrl_play.png mute.png cam.png skinconfig
 
 debug:
 install-debug: install
diff -urN xine-ui-0.9.4/src/xitk/skins/pitt/Makefile.am xine-ui-0.9.4.new/src/xitk/skins/pitt/Makefile.am
--- xine-ui-0.9.4/src/xitk/skins/pitt/Makefile.am	Tue Nov  6 23:53:01 2001
+++ xine-ui-0.9.4.new/src/xitk/skins/pitt/Makefile.am	Wed Nov  7 00:10:46 2001
@@ -11,7 +11,7 @@
 	playlist.png setup.png mrl.png  pl_play.png pl_delete.png \
 	pl_deleteall.png pl_moveup.png pl_movedown.png pl_inputtxt.png \
 	playfast.png playslow.png mrl_play.png volumebg.png volumepad.png mute.png \
-	cam.png sk_itembtn.png sk_slidv.png skinconfig
+	cam.png skinconfig
 
 debug:
 install-debug: install
 mkinstalldirs = $(SHELL) $(top_srcdir)/mkinstalldirs
diff -urN xine-ui-0.9.4/src/xitk/skins/xinetic/Makefile.am xine-ui-0.9.4.new/src/xitk/skins/xinetic/Makefile.am
--- xine-ui-0.9.4/src/xitk/skins/xinetic/Makefile.am	Tue Nov  6 23:53:01 2001
+++ xine-ui-0.9.4.new/src/xitk/skins/xinetic/Makefile.am	Wed Nov  7 00:11:22 2001
@@ -12,8 +12,7 @@
 	PlDeleteAll.png SmallArrowDown.png MainAutoButton.png MainPrev.png \
 	PlDismiss.png SmallArrowUp.png MainBg.png MainStop.png PlInputTxt.png \
 	ArrowUp.png ArrowDown.png MainVolumeBg.png MainVolumePad.png MainMute.png \
-	MainCam.png MainUp.png MainDown.png SkItemBtn.png SkSlidBG.png SkSlidPad.png \
-	SkArrowUp.png SkArrowDown.png skinconfig
+	MainCam.png MainUp.png MainDown.png skinconfig
 
 debug:
 install-debug: install
