--- xine-ui-0.9.21/configure.ac.orig	2003-05-20 21:00:16.000000000 +0200
+++ xine-ui-0.9.21/configure.ac	2003-05-20 21:12:06.000000000 +0200
@@ -634,8 +634,7 @@
         AC_TRY_CFLAGS("$sarchopt=athlon", k7cpu="athlon", k7cpu="i686")
 
 	dnl add x86 specific CFLAGS
-	CFLAGS="$CFLAGS -O3 -pipe -fomit-frame-pointer $f_af $f_al $f_aj $m_wm $m_psb -fexpensive-optimizations $f_si $f_nsa -ffast-math -funroll-loops -finline-functions"
-	DEBUG_CFLAGS="$DEBUG_CFLAGS -O"
+	CFLAGS="$CFLAGS -pipe -fomit-frame-pointer $f_af $f_al $f_aj $m_wm $m_psb -fexpensive-optimizations $f_si $f_nsa -ffast-math"
 	
 	dnl enable x86 specific parts of the code
 
@@ -670,17 +669,9 @@
 		archopt_val=athlon ;;
 	    esac
 
-	    if test x"$archopt_val" != x; then
-		CFLAGS="$CFLAGS $sarchopt=$archopt_val"
-		DEBUG_CFLAGS="$DEBUG_CFLAGS $sarchopt=$archopt_val"
-	    fi
 	]
 	fi
 
-    else
-	dnl add x86 specific cc CFLAGS
-	CFLAGS="$CFLAGS -O"
-	DEBUG_CFLAGS="$DEBUG_CFLAGS -O"
     fi
 
     AC_DEFINE(__i386__,1,[Define this if you're running x86 architecture])
@@ -689,57 +680,39 @@
 
   alphaev56-*)
     AC_DEFINE(__alpha__,1,[Define this if you're running Alpha architecture])
-    CFLAGS="$CFLAGS -O3 -mcpu=ev56 -mieee"
-    DEBUG_CFLAGS="$DEBUG_CFLAGS -O -mcpu=ev56 -mieee"
+    CFLAGS="$CFLAGS -mcpu=ev56 -mieee"
+    DEBUG_CFLAGS="$DEBUG_CFLAGS -mcpu=ev56 -mieee"
     ;;
 
   alpha*)
     AC_DEFINE(__alpha__,1,[Define this if you're running Alpha architecture])
-    CFLAGS="$CFLAGS -O3 -mieee"
-    DEBUG_CFLAGS="$DEBUG_CFLAGS -O -mieee"
+    CFLAGS="$CFLAGS -mieee"
+    DEBUG_CFLAGS="$DEBUG_CFLAGS -mieee"
     ;;
 
   ppc-*-linux-* | powerpc-*) 
     AC_DEFINE(__ppc__,1,[Define this if you're running PowerPC architecture])
-    CFLAGS="$CFLAGS -O3 -pipe -fomit-frame-pointer $m_wm $m_psb -fexpensive-optimizations $f_si $f_nsa -ffast-math -funroll-loops -funroll-all-loops -finline-functions"
-    DEBUG_CFLAGS="$DEBUG_CFLAGS -O"
+    CFLAGS="$CFLAGS -pipe -fomit-frame-pointer $m_wm $m_psb -fexpensive-optimizations $f_si $f_nsa -ffast-math -funroll-loops -funroll-all-loops -finline-functions"
     ;;
 
   sparc64-*-linux-* | sparc-*)
     AC_DEFINE(__sparc,1,[Define this if you're running Sparc architecture])
 
-    if test "$GCC" = yes; then
-	CFLAGS="$CFLAGS -O3"
-	DEBUG_CFLAGS="$DEBUG_CFLAGS -O"
-    else
-	CFLAGS="$CFLAGS -O"
-	DEBUG_CFLAGS="$DEBUG_CFLAGS -O"
-    fi
     ;;
 
   mips-*)
-    CFLAGS="$CFLAGS -O3"
-    DEBUG_CFLAGS="$DEBUG_CFLAGS -O"
     ;;
 
   m68k-*)
-    CFLAGS="$CFLAGS -O2"
-    DEBUG_CFLAGS="$DEBUG_CFLAGS -O"
     ;;
 
   ia64-*)
-    CFLAGS="$CFLAGS -O3"
-    DEBUG_CFLAGS="$DEBUG_CFLAGS -O"
     ;;
 
   hppa*-hp-*)
-    CFLAGS="$CFLAGS -O2"
-    DEBUG_CFLAGS="$DEBUG_CFLAGS -O"
     ;;
 
   s390-*)
-    CFLAGS="$CFLAGS -O3"
-    DEBUG_CFLAGS="$DEBUG_CFLAGS -O"
     ;;
 
   *)
@@ -755,11 +728,6 @@
     echo "your compiler to <xine-devel@lists.sf.net>"\!
     echo
     echo "*********************************************************************"
-    echo "(sleeping one minute so you can read this...)"
-    sleep 60
-
-    CFLAGS="$CFLAGS -O3"
-    DEBUG_CFLAGS="$DEBUG_CFLAGS -O"
     ;;
 esac
 
